dspace/config/registries/schema-project-types.xml

```diff
@@ -16,5 +16,28 @@
         - project.identifier.startdate > NOT FOUND
         - project.identifier.expectedcompletion > NOT FOUND
     -->
+	
+	<!--
+	  OpenAIRE4 Guidelines
+      4.2.2. Subproperty funderName (M)
+      Name of the funding provider (occurrence: 1). Mandatory if FundingReference is used. -->
+    <dc-type>
+        <schema>project</schema>
+        <element>funder</element>
+		<element>name</element>
```

Comment:
Shouldn't it be <element>funderName</element>

Reply:
Thank you for your suggestion Dimitris! 
According to https://schema.org/Project funder is described as:

> A person or organization that supports (sponsors) something through some kind of financial contribution.

The representation is Organization or Person and each of these have a "name" field. Since we have decided to use Schema.org for entities (https://wiki.lyrasis.org/display/DSPACE/2019-06-18+DSpace+7+Entities+WG+Meeting), `funderName` has a representation in Schema.org as `project.funder.name`.